Tasks/AzureFileCopy/AzureUtilityLTE9.8.ps1

```diff
@@ -83,6 +83,43 @@ function Create-AzureStorageContext
     }
 }
 
+function Get-AzureBlobStorageEndpointFromRDFE
+{
+    param([string]$storageAccountName)
+
+    if(-not [string]::IsNullOrEmpty($storageAccountName))
+    {
+        Switch-AzureMode AzureServiceManagement
+
+        Write-Verbose "[Azure Call](RDFE)Retrieving storage account endpoint for the storage account: $storageAccount"
+        $storageAccountInfo = Get-AzureStorageAccount -StorageAccountName $storageAccountName -ErrorAction Stop
+        $stoargeAccountEnpoint = $storageAccountInfo.Endpoints[0]
+        Write-Verbose "[Azure Call](RDFE)Retrieved storage account endpoint successfully for the storage account: $storageAccount"
+
+        return $stoargeAccountEnpoint
```

Comment:
You can fix the spelling of this variable. Ensure it is consistent in all place.